modules/cnf-about-topology-aware-lifecycle-operator-config.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: CONCEPT
+[id="cnf-about-topology-aware-lifecycle-operator-config_{context}"]
+= About the Topology Aware Lifecycle Operator configuration
+
+The Topology Aware Lifecycle Operator manages the deployment of update policies for one or more SNO clusters in a given duration. The Operator allows you to control the update order of {rh-rhacm}-managed SNO clusters in succession.
```

Comment:
The Topology Aware Lifecycle Operator manages the deployment of policies for one or more SNO clusters in a given time interval. The Operator allows you to control the update order of {rh-rhacm}-managed SNO clusters.